test/unit/team-users-test.js

```diff
@@ -0,0 +1,50 @@
+import Axios from 'axios'
+import { expect } from 'chai'
+import { describe, it } from 'mocha'
+import MockAdapter from 'axios-mock-adapter'
+import { TeamUsers } from '../../lib/organization/team/teamUsers'
+import { teamUsersMock, noticeMock } from './mock/objects'
+
+describe('Contentstack Team Users test', () => {
+  it('should query and find all users', done => {
+    var mock = new MockAdapter(Axios)
+    mock.onGet(`/organizations/organization_uid/teams/team_uid/users`).reply(200, teamUsersMock)
+    makeTeamUsers().query().find()
```

Comment:
In the test case, I dont see any other function of query than find. Add test cases for other functions if query is required, else we can replace query.find with fetchAll